### ISSUE TITLE
Output for R75pTOT/R95pTOT/R99pTOT is empty for CCAM/BARPA/CMIP6 models

### ISSUE THREAD

**Issue by ngben** [AusClimateService/indices#13]
icclim isn't outputting data for R75pTOT/R95pTOT/R99pTOT for CCAM/BARPA or CMIP6 models. It worked fine for AGCDv1. I am wondering if this is caused by unit differences? AGCDv1 is `mm` and CCAM/CMIP6/BARPA is `kg m-2 s-1`.

This plot is from `/g/data/ia39/australian-climate-service/test-data/CORDEX-CMIP6/indices/AUS-15/BOM/ECMWF-ERA5/evaluation/none/BOM-BARPA-R/v1/climdex/R75pTOT/R75pTOT_AUS-15_ECMWF-ERA5_evaluation_v1_month_19850101-20141231.nc`
![image](https://user-images.githubusercontent.com/34051150/218590399-2fa2bbe1-40be-40d6-92dd-ee78b5615471.png)

This plot is from `/g/data/ia39/australian-climate-service/test-data/CORDEX-CMIP6/indices/AUS-r005/none/BOM-AGCD/historical/none/none/none/climdex/R99pTOT/R99pTOT_AUS-r005_BOM-AGCD_historical_none_month_19850101-20141231.nc`
![image](https://user-images.githubusercontent.com/34051150/218590660-01e9e3f7-12af-4ed0-80ae-e7d835a718ef.png)

**Comment by DamienIrving** [comment]
This appears to be a problem just with the `FractionOfTotal` indicators - I've logged an issue with the developers: https://github.com/cerfacs-globc/icclim/issues/256

Would you like me to implement a work around in `run_icclim.py` in the meantime?

**Comment by ngben** [comment]
Thanks Damien, we can let icclim fix it.

**Comment by DamienIrving** [comment]
@ngben According to the icclim developers this is now fixed in icclim 6.3.0.

**Comment by ngben** [comment]
Thanks Damien, I don't think this has been fixed? 
I recalculated using icclim 6.3.0 for BARPA ERA5 here: `/g/data/xv83/users/bxn599/ACS/icclim_indices/AUS-15/BOM/ECMWF-ERA5/evaluation/none/BOM-BARPA-R/v1/climdex/R99pTOT`
Can you please double check? I used my own conda environment so I'm not sure if that could be the issue. It's using icclim 6.3.0 from the conda repo

Output from `R99pTOT_AUS-15_ECMWF-ERA5_evaluation_none_BOM-BARPA-R_v1_month_19850101-20141231.nc`
```
print(ds['R99pTOT'].values)
[[[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]]

 [[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]]

 [[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]]

 ...

 [[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [inf inf inf ... nan nan nan]
  [inf inf inf ... nan nan nan]
  [inf inf inf ... nan nan nan]]

 [[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [inf inf inf ... nan nan inf]
  [inf inf inf ... nan nan inf]
  [nan nan nan ... nan nan inf]]

 [[nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  ...
  [nan nan nan ... nan nan nan]
  [nan nan nan ... nan nan nan]
  [nan nan inf ... nan nan inf]]]
  ```
